<launch>
  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find manipulator_pose_following)/config/logger.conf"/>
  <node pkg="manipulator_pose_following" type="pose_following"
        name="pose_following" output="screen">
    <rosparam>
      rate: 40
      group: 'kuka_full'
      w0: 0.01
      k0: 0.01
      theta_d_lim: 3.14
      dt_pose_lim: 0.5
    </rosparam>
  </node>
</launch>
